static/searchPage.html

```diff
@@ -3,8 +3,9 @@
   <head>
     <meta charset="UTF-8" />
     <title>Ogene - Group Search</title>
-    <link rel="stylesheet" href="/css/searchPage.css" />
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDDhfcuk15Apx3i72mFSilulsPtJReGhcY"></script>
```

Comment:
Has this API key been properly restricted, or guarded in some other manner?

Reply:
I restricted the API to those we decided to use: Maps and Places. Is that enough?

Reply:
Sorry, I should have been more clear - I meant restricting the API key to prevent abuse, see https://developers.google.com/maps/documentation/embed/get-api-key#restrict_key for more info.

Reply:
So, I just restricted the API usage to our deployed web app and local port

Reply:
![image](https://user-images.githubusercontent.com/43916915/87728297-c4407c80-c790-11ea-8538-bd38abe17fb3.png)

These are the websites I restricted the API to. Right now Asha, Ved, and I aren't in sync with the way we run the app locally. We could always edit the restricted websites when we get that resolved.